marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/graphValidations.spec.tsx

```diff
@@ -93,7 +93,7 @@ describe("Graph Validations", () => {
     graphViewSidePanel.getDeleteIcon("BabyRegistry").should("exist");
   });
 
-  it("can center on entity type in graph view", {defaultCommandTimeout: 120000}, () => {
+  it.skip("can center on entity type in graph view", {defaultCommandTimeout: 120000}, () => {
```

Comment:
@lucianaabc-ml any reason for skipping this test ?

Reply:
Yes, because with the graph dropdown commented, this functionality is not available so this test was breaking.

Reply:
@lucianaabc-ml Please add a comment that it should be handled as a part of DHFPROD-8284